plugins/templates/README.md

```diff
@@ -83,7 +83,16 @@ Example payload:
 ```
 
 #### Template Tag
-Template tag allows fetching on template version by tag. For example, tag `mytemplate@1.1.0` as `stable`. 
+Template tag allows fetching on template version by tag. For example, tag `mytemplate@1.1.0` as `stable`.
+
+##### Delete a template
```

Comment:
Actually, the `##### Delete a template` section should probably go under the `#### Template` section rather than `#### Template Tag` section.